changelogs/unreleased/gh-6085-log-and-limit-iteration.md

```diff
@@ -0,0 +1,6 @@
+## feature/core
+
+* **[Breaking change]** Introduced fiber slice - time quota
+  for running fiber without yields. Use fiber slice in box:
+  throw an error in select/get/pairs/DML methods if fiber
+  slice is over (gh-6085).
```

Comment:
Missing a [footnote](https://github.com/tarantool/tarantool/blob/master/doc/changelogs.md#leave-a-note-to-a-release-manager). Also, "Use fiber slice in box" sounds awkward. I suggest to rewrite as follows:

```
 * **[Breaking change]** Introduced the mechanism for catching fibers running
   without yield for too long. Now, box operations, such as `select` and
   `replace`, will throw an error if the fiber execution time since yield
   exceeds the limit. The limit can also be checked from the application code
   with `fiber.check_slice()`. The default limit is one second, but it can be
   overwritten with `fiber.set_slice()` (gh-6085).

----

Breaking change: long running non-yielding fibers can be interrupted because of
the fiber execution time slice limit.
```

Also, please mention the default limit in the doc bot request.

Reply:
done